segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/IndexAppendProcessor.java

```diff
@@ -40,39 +40,52 @@ public class IndexAppendProcessor {
     static final Duration TIMEOUT = Duration.ofMinutes(1);
     private final ScheduledExecutorService indexSegmentUpdateExecutor;
     private final StreamSegmentStore store;
+    private final LatestItemSequentialProcessor<IndexAppend> latestItemProcessor;
```

Comment:
This should really be a `MultiKeyLatestItemSequentialProcessor` where the key is the segment.

Reply:
Changed to MultiKeyLatestItemSequentialProcessor.